www/netkisdk.js

```diff
@@ -1,4 +1,3 @@
-cordova.define("cordova-plugin-netkisdk.netkisdk", function(require, exports, module) {
```

Comment:
Why did you delete that?
Did you test that TS and JS files are able to find the plugin when removing this line.